tests/fiaas_deploy_daemon/deployer/kubernetes/test_ready_check.py

```diff
@@ -114,6 +116,39 @@ def test_deployment_complete_deactivated(self, get, app_spec, bookkeeper, lifecy
         lifecycle.success.assert_called_with(lifecycle_subject)
         lifecycle.failed.assert_not_called()
 
+    def test_ingress_ready(self, get, app_spec, bookkeeper, lifecycle, lifecycle_subject, config):
+        config.tls_certificate_ready = True
+        replicas = 2
+
+        with mock.patch("k8s.models.ingress.Ingress.find") as find:
+            ingress = mock.create_autospec(Ingress)
+            ingress.spec.tls.secretName = "secret1"
+            find.return_value = [ingress]
+
+            with mock.patch("k8s.models.custom_resource_definition.CustomResourceDefinition.get") as get:
+                cert = mock.create_autospec(CustomResourceDefinition)
+                condition = mock.create_autospec(CustomResourceDefinitionCondition)
+                condition.type = "Ready"
+                condition.status = "True"
+                cert.status.conditions = [condition]
+
+            self._create_response(get, replicas, replicas, replicas, replicas)
+            ready = ReadyCheck(app_spec, bookkeeper, lifecycle, lifecycle_subject, config)
+
+            assert ready() is True
+
+    def test_ingress_not_ready(self, get, app_spec, bookkeeper, lifecycle, lifecycle_subject, config):
+        config.tls_certificate_ready = True
+        replicas = 2
+
+        with mock.patch("k8s.models.ingress.Ingress.find") as find:
+            find.return_value = []
+
+            self._create_response(get, replicas, replicas, replicas, replicas)
+            ready = ReadyCheck(app_spec, bookkeeper, lifecycle, lifecycle_subject, config)
+
+            assert ready() is False
+
```

Comment:
Add test to ensure that, for the case of ingress without tls, the method returns true